apps/trustlab/src/payload/blocks/PageHeader.js

```diff
@@ -4,9 +4,17 @@ const PageHeader = BannerBlock(
   "page-header",
   "/images/cms/blocks/page-header.png",
   {
-    hooks: {
-      afterRead: [({ doc }) => ({ ...doc, isPageHeader: true })],
-    },
+    fields: [
+      {
+        name: "isPageHeader",
+        type: "checkbox",
+        defaultValue: true,
+        virtual: true,
+        admin: {
+          hidden: true,
```

Comment:
Not exactly sure why we are introducing a field but have it hidden again.

Reply:
🤔 ... yeah, it's bummer if we can't have hooks on blocks (I assume we can still add a field hook on the collection using these blocks that will mean repeating the hooks on each collection).

Stepping back for a bit, our implementation of `BannerBlock` mandates that each custom banner passes a `slug` i.e. `blockType`. If we already have this on the frontend, that is we can tell these banners apart, do we really need anything extra?

Reply:
@kilemensi That's the simplest option. Let me use the `slug` to differentiate the different banners.